test/util.ts

```diff
@@ -142,7 +143,7 @@ describe('Utils', () => {
         replicaset: {
           name: 'x'
         }
-      }).should.throw('`replicaset.members` is not set or is not an array');
+      } as unknown as MongoConfig).should.throw('`replicaset.members` is not set or is not an array');
```

Comment:
Ah, the legendary double cast.  

I've had to do this / have seen this more then I'd like to recently.